### ISSUE TITLE
Add “play” or “execute” button to presets

### ISSUE THREAD

**Issue by JanGVH** [glikely/obs-ptz#121]
When choosing another preset I have to double-click.
this is difficult with a quick scene change.
can that be a single click?

**Comment by rasamassen** [comment]
Agreed. A single-click option or default would be so much better. I don't have to double-click the camera movement controls. Why do I have to double-click the presets?

**Comment by markusd1984** [comment]
This may work as a safety mechanism so you don't accidentally change the camera preset/view which would be quite noticeable during a live stream, thus probably worthwhile the extra step and keeping it. 

Perhaps we could have a smaller button next to the preset for instant switching or a setting to change from double click to single click?

**Comment by glikely** [comment]
Yes, the double click is because you would also click on a preset to edit it without activating it. I don’t want to break that workflow. However, I like the idea of adding an activate button on the preset row

**Comment by yourgeeman** [comment]
Is this still in the works? Need it as a button option please!
Thanks!

**Comment by JanGVH** [comment]
@glikely first a big THANK YOU for all the effort You put in it. happily using your ptz control.
if I could use half the number of clicks it would be working faster when quickly moving between presets.
may the simplest way is to make it a setting, choosing between single or double click to activate a move to a preset position.

**Comment by glikely** [comment]
Now implemented in main. Will be part of v0.20.x major release
